### ISSUE TITLE
[Bug] Only reading 240 mb usage from main drive which has more like 30 GB usage

### ISSUE THREAD

**Issue by Spillebulle** [MauriceNino/dashdot#594]
### Description of the bug

![image](https://user-images.githubusercontent.com/46653946/218658479-4d7c8464-07a9-4b50-97b9-811e7ea87e04.png)
As shown in the image the main drive according to dash. only has 246 mb used, when in reality it has over 30 GB. The second drive shows perfectly though.

### How to reproduce

Install dashdot in docker with / bound to /mnt/host in the container

### Info output of dashdot cli

```shell
yarn run v1.22.19
$ node dist/apps/cli/main.js info
INFO
=========
Yarn: 1.22.19
Node: v18.14.0
Dash: 0.0.0-dev

Cwd: /app
Hash: 82b65f9d2b62dcadbd3e00f3e91fdbf0739e4a1d
In Docker: true
In Podman: false
In Docker (env): true
Done in 0.70s.
```

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### Where is your instance running?

Linux Server

**Comment by MauriceNino** [comment]
Hi there, thanks for creating an issue!

Can you please post the output of the following command?

```
docker exec CONTAINER yarn cli raw-data --storage
```

**Comment by Spillebulle** [comment]
> Hi there, thanks for creating an issue!
> 
> Can you please post the output of the following command?
> 
> ```
> docker exec CONTAINER yarn cli raw-data --storage
> ```
yarn run v1.22.19
$ node dist/apps/cli/main.js raw-data --storage
If you were asked to paste the output of this command, please post only the following:

- On GitHub: Everything between (and excluding) the lines
- On Discord: Everything between (and including) the ```

----------------------------------------

<details>
  <summary>Output:</summary>

```js
const disks =  [
  {
    device: '/dev/sda',
    type: 'HD',
    name: 'MR9240-8i       ',
    vendor: 'LSI',
    size: 291999055872,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '2.13',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  },
  {
    device: '/dev/sdb',
    type: 'HD',
    name: 'MR9240-8i       ',
    vendor: 'LSI',
    size: 896998047744,
    bytesPerSector: null,
    totalCylinders: null,
    totalHeads: null,
    totalSectors: null,
    totalTracks: null,
    tracksPerCylinder: null,
    sectorsPerTrack: null,
    firmwareRevision: '2.13',
    serialNum: '',
    interfaceType: '',
    smartStatus: 'unknown',
    temperature: null
  }
]
const sizes =  [
  {
    fs: 'overlay',
    type: 'overlay',
    size: 105089261568,
    used: 35669483520,
    available: 64034291712,
    use: 35.78,
    mount: '/',
    rw: false
  },
  {
    fs: '/dev/mapper/ubuntu--vg-ubuntu--lv',
    type: 'ext4',
    size: 105089261568,
    used: 35669483520,
    available: 64034291712,
    use: 35.78,
    mount: '/mnt/host',
    rw: true
  },
  {
    fs: '/dev/loop1',
    type: 'squashfs',
    size: 66453504,
    used: 66453504,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1778',
    rw: false
  },
  {
    fs: '/dev/loop2',
    type: 'squashfs',
    size: 108003328,
    used: 108003328,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/23541',
    rw: false
  },
  {
    fs: '/dev/loop4',
    type: 'squashfs',
    size: 52035584,
    used: 52035584,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/17883',
    rw: false
  },
  {
    fs: '/dev/sdb2',
    type: 'fuseblk',
    size: 896979169280,
    used: 107469594624,
    available: 789509574656,
    use: 11.98,
    mount: '/mnt/host/mnt/Pancake-Data',
    rw: true
  },
  {
    fs: '/dev/sda2',
    type: 'ext4',
    size: 2040373248,
    used: 258015232,
    available: 1658208256,
    use: 13.46,
    mount: '/mnt/host/boot',
    rw: true
  },
  {
    fs: '//192.168.0.2/SegaMegaDrive',
    type: 'cifs',
    size: 32001783488512,
    used: 21775525765120,
    available: 10226257723392,
    use: 68.04,
    mount: '/mnt/host/mnt/SegaMegaDrive',
    rw: true
  },
  {
    fs: '192.168.0.6:/mnt/SegaGenesis/SegaGenesisDataset',
    type: 'nfs',
    size: 52162231795712,
    used: 40259038871552,
    available: 11903192924160,
    use: 77.18,
    mount: '/mnt/host/mnt/SegaGenesis',
    rw: false
  },
  {
    fs: '/dev/loop5',
    type: 'squashfs',
    size: 52297728,
    used: 52297728,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/snapd/17950',
    rw: false
  },
  {
    fs: '/dev/loop6',
    type: 'squashfs',
    size: 117440512,
    used: 117440512,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/lxd/24322',
    rw: false
  },
  {
    fs: '/dev/loop3',
    type: 'squashfs',
    size: 66453504,
    used: 66453504,
    available: 0,
    use: 100,
    mount: '/mnt/host/snap/core20/1822',
    rw: false
  }
]
const blocks =  [
  {
    name: 'sda',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 291999055872,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'MR9240-8i',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sdb',
    type: 'disk',
    fsType: '',
    mount: '',
    size: 896998047744,
    physical: 'HDD',
    uuid: '',
    label: '',
    model: 'MR9240-8i',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sdb'
  },
  {
    name: 'loop1',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1778',
    size: 66347008,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'loop2',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/23541',
    size: 107986944,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'loop3',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/core20/1822',
    size: 66355200,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'loop4',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/17883',
    size: 52031488,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'loop5',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/snapd/17950',
    size: 52248576,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'loop6',
    type: 'loop',
    fsType: 'squashfs',
    mount: '/mnt/host/snap/lxd/24322',
    size: 117387264,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: ''
  },
  {
    name: 'sda1',
    type: 'part',
    fsType: '',
    mount: '',
    size: 1048576,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sda2',
    type: 'part',
    fsType: 'ext4',
    mount: '/mnt/host/boot',
    size: 2147483648,
    physical: '',
    uuid: 'f1639040-f8e2-40db-ba49-40458274a014',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sda3',
    type: 'part',
    fsType: 'LVM2_member',
    mount: '',
    size: 289848426496,
    physical: '',
    uuid: 'mN6Lat-6d5z-R3MI-F4XP-uLFQ-LNAH-5MvmOV',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sda'
  },
  {
    name: 'sdb1',
    type: 'part',
    fsType: '',
    mount: '',
    size: 16759808,
    physical: '',
    uuid: '',
    label: '',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sdb'
  },
  {
    name: 'sdb2',
    type: 'part',
    fsType: 'ntfs',
    mount: '/mnt/host/mnt/Pancake-Data',
    size: 896979173376,
    physical: '',
    uuid: 'E23E57F13E57BCEB',
    label: 'Data',
    model: '',
    serial: '',
    removable: false,
    protocol: '',
    group: '',
    device: '/dev/sdb'
  }
]
```

</details>

----------------------------------------
Done in 0.63s.

**Comment by MauriceNino** [comment]
:tada: This issue has been resolved in version 4.9.1

Please check the [changelog](https://github.com/MauriceNino/dashdot/blob/main/.github/CHANGELOG.md) for more details.
